cmd/tsdbrelay/doc.go

```diff
@@ -22,15 +22,16 @@ Usage:
 
 The flags are:
 	-b="bosun"
-		Target Bosun server. Can specify port with host:port.
+		Target Bosun server. Can specify as host, host:port, or https://host:port.
 	-t=""
-		Target OpenTSDB server. Can specify port with host:port.
+		Target OpenTSDB server. Can specify as host, host:port or https://host:port.
 	-l=":4242"
 		Listen address.
 	-v=false
 	    Enable verbose logging
 	-r=""
 		Additional relays to send data to, comma seperated. Intended for secondary data center replication. Only response from primary tsdb server wil be relayed to clients.
+		Examples: hostA:port,https://hostB:port,hostC#data-only,https://hostD:8080#metadata-only
```

Comment:
The fragment thing feels super dirty. But I can't think of a better way immediately. Are we outgrowing the capacity to configure this thing without a more complicated config?

```
{
"primaryTsdb": "ny-tsdb03",
"primaryBosun": "ny-bosun01",
"relays":[
  {"type":"tsdb", "host":"ny-tsdbcluster2"},
  {"type":"tsdbrelay","host":"co-tsdb01"},
  {"type":"bosun","host":"ny-bosunxy42"}
 ]
}
```

Is starting to be more appealing to me than super-dense flags. Or whatever the toml equivalent is.

Something to think about.